source/deployment/customize-mattermost.rst

```diff
@@ -0,0 +1,59 @@
+
+If customizing Mattermost, please avoid branding that could be confused with the Mattermost brand. For example, it's okay to brand as "Healthcare Central" because it's a completely different brand. "Mattermost Healthcare Central" is not okay, because it can potentially be confused with the Mattermost brand. Please see the `Mattermost trademark guidelines <https://www.mattermost.org/trademark-standards-of-use/>`_ for details.
+
+While you're welcome to add your own copyright notice in the user interface if you feel it is warranted by your changes, we ask that you do not remove the Mattermost, Inc. copyright notice from the login footer or from the About dialog.
+
+Mattermost Webapp
+-----------------
+
+The Mattermost webapp is licensed under the Apache 2.0 license. To modify and use with the Mattermost server, you can:
+
+1. Install the Mattermost server by following one of our installation guides
+2. Fork the `mattermost-webapp <https://github.com/mattermost/mattermost-webapp>`_ repository
+3. Make your changes 
+4. Run ``make package``. This will create ``mattermost-webapp.tar.gz``. 
+5. Copy ``mattermost-webapp-tar.gz`` to the location Mattermost was installed in Step 1. 
+6. Remove the existing ``client`` folder. 
+7. Run ``tar -xvf mattermost-webapp.tar.gz`` to extract your new customized ``client`` folder
+8. Restart your Mattermost server 
+
+In Mattermost Enterprise Edition, it is possible to customize certain parts of the webapp without forking using our :doc:`Custom Branding <../administration/branding>` settings. 
+
+Mattermost Server
+-----------------
+
+There are a few things that can be customized in the Mattermost server without forking. 
+
+1. Modify text in the Mattermost interface by modifying the ``en.json`` file 
+2. Replace the logo in email notifications by changing the file located in the ``/images`` directory 
+3. Customize or hide help and support links by modifying your :ref:`configuration settings <legal-support-links>`
+
+Mattermost Mobile Applications
+------------------------------
+
+The Mattermost mobile applications can be customized if you choose to build the apps yourself. 
+
+To brand the mobile apps: 
+
+1. Fork the `mattermost-mobile <https://github.com/mattermost/mattermost-mobile>`_ repository
+2. Replace the name, images, and any key text strings
+3. Compile the apps
+4. Deploy the apps to an app store
+
+While most organizations deploy to internal enterprise app stores, you are welcome to deploy to iTunes and Google Play as long as the branding is not confusable with official Mattermost products.
+
+Mattermost Desktop Applications
+-------------------------------
+
+The Mattermost desktop applications can be customized if you choose to build the apps yourself. 
+
+To brand the desktop apps: 
+
+1. Fork the `mattermost/desktop <https://github.com/mattermost/desktop>`_ repository
+2. Replace the name, images, and any key text strings
+3. Compile the apps
```

Comment:
@jasonblais not sure if there's documentation we can link to about compiling desktop app, or anything else about desktop app deployment that would be useful to link here

Reply:
We could link to https://github.com/mattermost/desktop/blob/master/docs/development.md